Use relative symlinks for view, ex

Create symlinks like:
  /usr/bin/ex -> vi
  /usr/bin/view -> vi

rather than:
  /usr/bin/ex -> /usr/bin/vi
  /usr/bin/view -> /usr/bin/vi

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -616,3 +616,3 @@
 	@$(VERBOSE); $(TEST) -x "$(PREFIX)/bin/$(BINPREFIX)vi$(BINSUFFIX)" && \
-        $(LNS) "$(PREFIX)/bin/$(BINPREFIX)vi$(BINSUFFIX)"                     \
+        $(LNS) "$(BINPREFIX)vi$(BINSUFFIX)"                                   \
             "$(PREFIX)/bin/$(BINPREFIX)ex$(BINSUFFIX)"
@@ -623,3 +623,3 @@
 	@$(VERBOSE); $(TEST) -x "$(PREFIX)/bin/$(BINPREFIX)vi$(BINSUFFIX)" && \
-        $(LNS) "$(PREFIX)/bin/$(BINPREFIX)vi$(BINSUFFIX)"                     \
+        $(LNS) "$(BINPREFIX)vi$(BINSUFFIX)"                                   \
             "$(PREFIX)/bin/$(BINPREFIX)view$(BINSUFFIX)"
